config/application.yml.default

```diff
@@ -112,6 +112,7 @@ irs_attempt_api_event_count_default: 1000
 irs_attempt_api_event_count_max: 10000
 kantara_2fa_phone_restricted: false
 kantara_2fa_phone_existing_user_restriction: false
+kantara_restriction_enforcement_date: '2022-07-19'
```

Comment:
Can we make it sooner by default so it's enabled in local dev?

Reply:
I think its fine because it will still be enabled in local dev for now, just they will see different content and the skip logic won't skip only one time.